static/src/stylesheets/module/_sudoku.scss

```diff
@@ -0,0 +1,55 @@
+.sudoku__grid {
+    outline: none;
+}
+
+.sudoku__cell rect {
+    fill: #ffffff;
+    cursor: pointer;
+}
+
+.sudoku__cell--not-editable .sudoku__cell-text {
+    font-weight: bold;
+}
+
+.sudoku__cell--highlighted rect {
+    fill: lighten(colour(crosswordAccent2), 30%);
+}
+
+.sudoku__cell--focussed rect {
+    fill: lighten(colour(crosswordAccent2), 10%);
+}
+
+.sudoku__cell--error .sudoku__cell-text {
+    fill: colour(error);
+}
+
+.sudoku__cell-text {
+    cursor: pointer;
+    font-size: 38px;
+    text-anchor: middle;
+    font-family: $f-sans-serif-text;
+}
+
+.sudoku__cell-jotting {
+    font-weight: 600;
+    cursor: pointer;
+    font-size: 10px;
+    text-anchor: middle;
+    font-family: $f-sans-serif-text;
+}
+
+.sudoku__button-background {
+    cursor: pointer;
+    fill: #d7d7d7;
+
+    .sudoku__button:hover & {
+        fill: lighten(colour(crosswordAccent2), 30%);
```

Comment:
You seem to be using this colour function a lot, shouldn't this just be the variable default? Especially considering I can't find uses of any of your new colour variables being used in their 'raw' state.